README.md

```diff
@@ -53,6 +53,9 @@ Grab the [latest version](https://github.com/newhinton/Round-Sync/releases/lates
 
 If you don't know which version to pick use ```roundsync-<version>-universal-release.apk```. Most devices run ARM 64 Bit, and 64 Bit devices often can also run the respective 32 bit version at lower performance. The app runs on any phone, tablet or TV with Android 7 or newer, as long as you have a touchscreen or mouse.
 
+[<img src="https://f-droid.org/badge/get-it-on.png"
+    alt="Get it on F-Droid"
+    height="80">](https://f-droid.org/packages/de.felixnuesse.extract)
```

Comment:
```suggestion
[<img src="https://f-droid.org/badge/get-it-on.png"
    alt="Get it on F-Droid"
    height="80">](https://f-droid.org/packages/de.felixnuesse.extract) [<img src="https://gitlab.com/IzzyOnDroid/repo/-/raw/master/assets/IzzyOnDroidButtonGreyBorder_nofont.png" alt="Get it on IzzyOnDroid" height="80">](https://apt.izzysoft.de/packages/de.felixnuesse.extract)
```

Reply:
Your badge doesn't have padding so it looks much larger than F-Droid badge. How did other dev solve this problem?

Reply:
Ah, apologies! I didn't consider yours having spacing added. Then maybe using [this one](https://gitlab.com/IzzyOnDroid/repo/-/raw/master/assets/IzzyOnDroid.png) instead? We've got several badges to choose from, I just checked to pick one with border to match yours.

![IoD badge with spacing](https://gitlab.com/IzzyOnDroid/repo/-/raw/master/assets/IzzyOnDroid.png)

Feel free to update my suggested change with it – after all, it was just a **suggested** change :wink: And thanks for pointing out!